libs/coin-framework/src/derivation.ts

```diff
@@ -198,6 +198,9 @@ const modes = Object.freeze({
     overridesDerivation: "44'/397'/0'/0'/<account>'",
     mandatoryEmptyAccountSkip: 1,
   },
+  icon: {
+    overridesDerivation: "44'/4801368'/0'/0'/<account>'",
```

Comment:
Why do you use `4801368`, please? Should it not be 74 https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
@KVNLS I fixed this, thank you

Reply:
@lvndry The “proper” BIP44 path is `4801368`, not `74` https://forum.icon.community/t/creating-a-standard-for-the-bip44-path-derivation-of-icon-addresses-discussion/2702